### PR TITLE
Adapt to new moveit_resources layout

### PR DESCRIPTION
**Pull request by v4hn**
part of the effort to split moveit_resources into a number of packages.

Related to https://github.com/ros-planning/moveit_resources/pull/36 and https://github.com/ros-planning/moveit/pull/2199

### CODE REVIEW

File: package.xml [hunk 1]
```diff
@@ -53,7 +53,7 @@
   <run_depend>tf2_eigen</run_depend>
   <run_depend>tf2_geometry_msgs</run_depend>
 
-  <test_depend>moveit_resources</test_depend>
+  <test_depend>moveit_resources_panda_moveit_config</test_depend>
```
Comment (rhaschke) [inline package.xml:56]: Should we switch to native panda_moveit_config here? There is no need to use moveit_resources here for testing, is it?

Comment (v4hn) [reply package.xml:56]: There is one tutorial on creating tests that uses our panda model from resources ([see here](https://github.com/ros-planning/moveit_tutorials/blob/e49e1b0ac4dcb646bcc9295bfe04261d664aa8f8/doc/tests/test/tests.cpp#L72)).
The rest is supposed to use the native robot.

**Comment by SansoneG** [comment]
@v4hn Is there anything still holding this PR back from getting merged?

**Comment by rhaschke** [comment]
Note, Travis will fail here until moveit_resources is released.

**Comment by v4hn** [comment]
Thanks for taking care of the fine details and the merge @rhaschke :sunflower:

**Pull request merged**